### PR TITLE
commons: support conda env #6833

### LINKED ISSUE #6833: Support conda env
Configuration now support virtualenv but not conda / mamba env.

PR coming

### PR DESCRIPTION
**Pull request by wiso**
Now common/config.py supports virtualenv. With this PR it supports also conda / mamba environments.

### CODE REVIEW

**Comment by wiso** [comment]
do I really need to add myself to the list of contributors for only 2 lines?

File: lib/rucio/common/config.py [hunk 1]
```diff
@@ -709,6 +709,7 @@ def get_config_dirs() -> list[str]:
     Returns all available configuration directories in order:
     - $RUCIO_HOME/etc/
     - $VIRTUAL_ENV/etc/
+    - $CONDA_ENV/etc
```
Comment (rdimaio) [inline lib/rucio/common/config.py]: AFAIK `CONDA_ENV` is not a default environment variable - did you mean `CONDA_PREFIX` here?

**Comment by rdimaio** [comment]
> do I really need to add myself to the list of contributors for only 2 lines?

It's not mandatory, but you can do it if you want

File: lib/rucio/common/config.py (hunk 1 above)
Comment (wiso) [reply lib/rucio/common/config.py]: ops! the code actually is correct

**Review by rdimaio — APPROVED** [review]
LGTM - the commits should be squashed into a single commit, but we can do that on our side when merging

**Pull request merged**